qcengine/programs/nwchem/harvester.py

```diff
@@ -217,18 +217,19 @@ def harvest_outfile_pass(outtext):
                 psivar[f'{cc_corr} CORRECTION ENERGY'] = mobj.group(1)
                 psivar[f'{cc_plain} CORRELATION ENERGY'] = mobj.group(2)
                 psivar[f'{cc_plain} TOTAL ENERGY'] = mobj.group(3)
-        #Other TCE
-        #mobj = re.findall(
-        #        r'^\s+' + r'Iterations converged' + r'\s*' +
-         #       r'^\s+' + r'(.*?)' + r' correlation energy / hartree' + r'\s+=\s*' + NUMBER + r'\s*'
-         #       r'^\s+' + r'(.*?)' + r' total energy / hartree' + r'\s+=\s*' + NUMBER + r'\s*$', 
-          #      outtext, re.MULTILINE | re. DOTALL)
+        #Process other TCE cases
+        for cc_name in [r'CISD', r'CISDT', r'CISDTQ', r'CCD', r'CCSD', r'CCSDT', r'CCSDTQ', r'LCCSD', r'LCCD']:
+            mobj = re.findall(
+                r'^\s+' + r'Iterations converged' + r'\s*' +
+                r'^\s+' + cc_name + r' correlation energy / hartree' + r'\s+=\s*' + NUMBER + r'\s*'+
```

Comment:
[this](https://github.com/MolSSI/QCEngine/pull/98/files#diff-c0d24fbdb771ad09450a93a96fe06099L209) preferred because it doesn't hardcode the spaces before `correlation energy /` the way the above does.